BraveWallet/Crypto/BuySwapSwap/SwapCryptoView.swift

```diff
@@ -91,72 +233,153 @@ struct SwapCryptoView: View {
-                .font(.title3.weight(.semibold))
-                .foregroundColor(Color(.braveLabel))
+              Text(Strings.Wallet.swapCryptoSlippageTitle)
+                .font(.subheadline)
               Spacer()
-              Text(verbatim: "0.0000")
-                .font(.footnote)
+              Text(formatSlippage())
                 .foregroundColor(Color(.secondaryBraveLabel))
+                .font(.subheadline.weight(.semibold))
+              Button(action: {
+                hideSlippage.toggle()
+              }) {
+                Image("wallet-dismiss")
+                  .renderingMode(.template)
+                  .resizable()
+                  .foregroundColor(Color(.secondaryBraveLabel))
+                  .frame(width: 12, height: 6)
+                  .rotationEffect(Angle.degrees(hideSlippage ? 0 : 180))
+                  .animation(.default)
+              }
+              .buttonStyle(PlainButtonStyle())
             }
-            .padding(.vertical, 8)
           }
-        }
-        .listRowBackground(Color(.secondaryBraveGroupedBackground))
-        Section(
-          header: WalletListHeaderView(title: Text(Strings.Wallet.swapCryptoAmountReceivingTitle))
-        ) {
-          TextField(Strings.Wallet.swapCryptoAmountPlaceholder, text: .constant(""))
-        }
-        .listRowBackground(Color(.secondaryBraveGroupedBackground))
-        Section(
-          header: Picker(Strings.Wallet.swapOrderTypeLabel, selection: $orderType) {
-            Text(Strings.Wallet.swapMarketOrderType).tag(OrderType.market)
-            Text(Strings.Wallet.swapLimitOrderType).tag(OrderType.limit)
+          .listRowBackground(Color(.secondaryBraveGroupedBackground))
+          Section(
+            header:
+              Button(action: {}) {
+                Text(Strings.Wallet.swapCryptoSwapButtonTitle)
+              }
+              .buttonStyle(BraveFilledButtonStyle(size: .normal))
+              .frame(maxWidth: .infinity)
+              .resetListHeaderStyle()
+              .padding(.top, hideSlippage ? -20 : 20)
+          ) {
+          }
+        } else {
+          Section {
+            VStack(alignment: .leading, spacing: 4.0) {
+              Text(Strings.Wallet.swapCryptoUnsupportNetworkTitle)
+                .font(.headline)
+              Text(String.localizedStringWithFormat(Strings.Wallet.swapCryptoUnsupportNetworkDescription, ethNetworkStore.selectedChain.chainName))
+                .font(.subheadline)
+                .foregroundColor(Color(.secondaryBraveLabel))
+            }
+            .padding(.vertical, 6.0)
+            .listRowBackground(Color(.secondaryBraveGroupedBackground))
           }
-            .pickerStyle(SegmentedPickerStyle())
-            .resetListHeaderStyle()
-            .listRowBackground(Color(.clear))
-        ) {
         }
-        .listRowBackground(Color(.secondaryBraveGroupedBackground))
       }
       .navigationTitle(Strings.Wallet.swap)
```

Comment:
Need a `.animation(.default, value: hideSlippage)` above this to animate the opacity change & sections below it moving up and down